### PR TITLE
Support switch between using wellknown mac or server mac addr 

### PR DESCRIPTION
**Pull request by lolyu**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [x] New feature
- [ ] Doc/Design
- [ ] Unit test

#### What is the motivation for this PR?
Add a new field `use_known_mac` to table `LINK_PROBER`:
* if `use_known_mac` is `enable`, linkmgrd will use known mac to probe(default behavior)
* if `use_known_mac` is `disable`, linkmgrd will use server mac address from netlink

#### How did you do it?
Add a new boolean attribute `mUseKnownMacActiveActive` to `MuxConfig` to represent this switch.
If `mUseKnownMacActiveActive` is `false`, the state machine will respond to server mac change.
else, the state machine will not respond to server mac change.

#### How did you verify/test it?
Pass unit test

### CODE REVIEW

File: src/common/MuxConfig.h [hunk 1]
```diff
@@ -316,6 +316,24 @@ class MuxConfig
...
     inline bool getIfEnableDefaultRouteFeature() {return mEnableDefaultRouteFeature;};
 
+    /**
+     * @method getUseKnownMacActiveActive
+     * 
+     * @brief check if use known mac to probe for active-active ports
+     * 
+     * @return true to use known mac to probe for active-active ports
+     */
+    inline bool getUseKnownMacActiveActive() { return mUseKnownMacActiveActive; };
```
Comment (zjswhhh) [inline src/common/MuxConfig.h:326]: Better named as `getIfUseKnownMacActiveActive` as return value is a `bool`?

File: src/common/MuxConfig.h [hunk 2]
```diff
@@ -316,6 +316,24 @@ class MuxConfig
...
+
+    /**
+     * @method setUseKnownMacActiveActive
+     * 
+     * @brief setter to set if use known mac to probe for active-active ports
+     * 
+     * @return none
+     */
+    inline void setUseKnownMacActiveActive(bool useKnownMacActiveActive) { mUseKnownMacActiveActive = useKnownMacActiveActive; };
```
Comment (zjswhhh) [inline src/common/MuxConfig.h:335]: If config is updated to `false`, will previously updated server mac be consumed? No?

File: src/common/MuxConfig.h (hunk 1 above)
Comment (lolyu) [reply src/common/MuxConfig.h]: Updated

File: src/common/MuxConfig.h (hunk 2 above)
Comment (lolyu) [reply src/common/MuxConfig.h:335]: Now mux port config will cache the last mac-address update from Netlink, and use that if `useKnownMac` is set to `false`

**Comment by yxieca** [comment]
Should be well-known mac address. And the well-known mac address is set to all 0s now?

**Comment by lolyu** [comment]
> Should be well-known mac address. And the well-known mac address is set to all 0s now?

Sure, will update the name. The `mKnownMacAddress` has an initial value as all 0s, but it will be assigned with generated well-known mac for active-active ports.

**Comment by lolyu** [comment]
@yxieca @zjswhhh , updated, please take a review again!

**Review by zjswhhh — APPROVED** [review]

**Pull request merged**